media-fonts/fantasque-sans-mono/fantasque-sans-mono-2019.11.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit font
+
+DESCRIPTION="Functional programming font designed with some handwriting-like fuzziness"
+HOMEPAGE="http://openfontlibrary.org/en/font/fantasque-sans-mono"
+SRC_URI="http://openfontlibrary.org/assets/downloads/${PN}/99bd6bf2755c7f6b5f5c62daf6807aea/${PN}.zip -> ${P}.zip"
```

Comment:
prefer using the github version as it has actual versioning scheme instead of dated  
https://github.com/belluzj/fantasque-sans

Reply:
this also gets rid of unzip.